### DISCUSSION TITLE
Orientation of Antenna and Transmitter

### DISCUSSION THREAD

**Discussion by jonathanuniberg** [NVlabs/sionna#542] [Q&A]
Hello, I am currently testing sionna rt and I understand that you are able to tilt the transmitter as you wish but I didn't really understood the orientation and position of the antennas in the antenna array. It seems that the position and the orientation of the antenna have no impact on the coverage map when changed. Can I only control the tilt of the transmitter, because the rotated.position() method in AntennaArray doesn't really do anything. Thank you very much in advance.

**Comment by SebastianCa** [comment]
Hi @jonathanuniberg,
Which [antenna pattern](https://nvlabs.github.io/sionna/api/rt.html#iso-pattern) are you using for your experiments? Could you please provide a small example that allows us to reproduce the issue?

**Reply by jonathanuniberg** [reply]
Hello in this example, the rotated.position() has no effect. 

```py
t_antenna = Antenna(pattern="tr38901", polarization="VH")
r_antenna = Antenna(pattern="iso", polarization="VH")

t_position = tf.Variable([[0, 0, 0]], dtype=tf.float32)  

r_position = tf.Variable([[0, 0, 0]], dtype= tf.float32)


orientation = tf.constant([np.pi/4, 0.0, 0.0], dtype=tf.float32)  


t_antenna_array = AntennaArray(
    antenna= t_antenna,
    positions=t_position
)

r_antenna_array = AntennaArray(
    antenna= r_antenna,
    positions=r_position
)

t_rotated_positions = t_antenna_array.rotated_positions(orientation)
r_rotated_positions = r_antenna_array.rotated_positions(orientation)

scene.tx_array = t_antenna_array
scene.rx_array = r_antenna_array


print("Rotated Positions (Transmitter):", t_rotated_positions.numpy())
print("Rotated Positions (Receiver):", r_rotated_positions.numpy())
```

**Reply by SebastianCa** [reply]
It appears that the rotation is being applied around the origin, which might be why you’re not observing any changes. To better assess the effect of the rotation, could you please try running the same code snippet with different `t_position` and `r_position` values, such as `[1, 0, 0]`? This should help demonstrate the impact of the rotation more clearly.

**Reply by jonathanuniberg** [reply]
Thank you very much! The rotated.positions() will give me a newly calculated output now, but visually it doesn't do that much, when I change the orientation of an antenna. I read the documentation but the antenna transmitter relationship isn't quite clear to me. When I change the transmitter orientation it works perfectly fine, but when I also want to change the antenna orientation there is are no noticeable changes. Are the transmitter and the antenna dependent or related in any way?

**Reply by SebastianCa** [reply]
What do you mean by "visually"? In the scene preview, tx/rx are typically visualized as dot?

A general remark: the `AntennaArray` positions are the relative positions of antenna elements in the array. Is it possible that you are looking for the position/orientation of the [RadioDevices](https://nvlabs.github.io/sionna/api/rt.html#radio-devices) (i.e., transmitter and receiver)?
